docs/api/api-methods.md

```diff
@@ -0,0 +1,189 @@
+# API Methods
+All API methods calls except for token generation are secured via [OAuth2](oauth.md).
+You need to provide token via `Authorization` header to access requested data like this: ```Authorization: "Bearer eyJ0eXAiOiJKV...s3SKg"```
+
+**Retrieve access token**
+----
+
+* **URL**
+
+  `/api/token`
+
+* **Method:**
+
+  `POST`
+  
+*  **Data Params**
+
+   **Required:**
+ 
+   `grant_type=client_credentials`
+   `client_id=[string]`
+   `client_secret=[string]`
+
+* **Success Response:**
+
+  * **Code:** 200 <br />
+     **Content:** 
+        ```
+        {
+            "token_type": "Bearer",
+            "expires_in": 3600,
+            "access_token": "eyJ0eXAiOiJKV...s3SKg"
+        }
+        ```
+ 
+* **Error Response:**
+
+  * **Code:** 401 UNAUTHORIZED <br />
+    **Content:** 
+    ```
+    {
+        "error": "invalid_client",
+        "message": "Client authentication failed"
+    }
+    ```
+
+  OR
+
+  * **Code:** 400 Bad Request <br />
+
+**Retrieve a product**
+----
+
+* **URL**
+
+  `/api/v1/products/{uuid}`
+
+* **Method:**
+
+  `GET`
+  
+*  **URL Params**
+
+   **Required:**
+ 
+   `uuid=[string]`
+
+* **Success Response:**
+
+  * **Code:** 200 <br />
+     **Content:** 
+        ```
+        [
```

Comment:
single product API call doesn't return an array, there is the object itsef only

plus there are some `¨` in this document

Reply:
Fixed